src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsPagesTest.java

```diff
@@ -1017,8 +1017,7 @@ private static void validatePropertySectionPropertyEx(String fileName, String se
         final String expectedTypeName;
         // SuppressWarningsHolder#aliasList is backed by a static (upper case) property.
         if ("SuppressWarningsHolder".equals(sectionName) && "aliasList".equals(propertyName)) {
-            expectedTypeName = "String[] in a format of comma separated attribute=value entries. "
-                + "The attribute is the fully qualified name of the Check and value is its alias.";
+            expectedTypeName = "String[]";
```

Comment:
This change is not expected. This looks like valuable information we are losing detailing the format of the property.

Reply:
I think type should be this simple array but yes, I moved this description on how we interpretate value of this type to property desription.

Reply:
https://checkstyle.org/checks/annotation/suppresswarningsholder.html#SuppressWarningsHolder
vs
![image](https://github.com/checkstyle/checkstyle/assets/812984/b4c92c15-2409-47b1-a4cb-74c37a8acdac)